src/lib/viewers/controls/icons/IconDrawing16.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+function IconDrawing16(props: React.SVGProps<SVGSVGElement>): JSX.Element {
```

Comment:
It looks like the height/width are `24` and the viewbox is `0 0 0 18`. What is the 16 in reference to?

Reply:
The original svg that I used had 16 in the title, but I think I should use 24 instead since that's the height/width of the icon.
https://git.dev.box.net/Box/design-assets/blob/master/Icon/Fill/AnnotationsDrawing16.svg